src/libraries/System.Security.AccessControl/tests/RawAcl/RawAcl_RemoveAce.cs

```diff
@@ -41,6 +41,13 @@ public static void TestRemoveAce(string sddl)
                 rawAcl.RemoveAce(index);
             });
 
+            //test remove at value too large
+            Assert.Throws<ArgumentOutOfRangeException>(() =>
```

Comment:
Assuming there's a paramName being specified:

```suggestion
            AssertExtensions.Throws<ArgumentOutOfRangeException>(
                "whateverTheParamNameIs",
                () =>
```

Reply:
Good idea! Added!